mass_mailing_event/models/event_state.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class EventState(models.Model):
+    _name = 'event.state'
+
+    name = fields.Char(required=True)
```

Comment:
Why do you need this? If required, you need to include a pre-init hook that fills the fields that are empty.